package.json

```diff
@@ -4,8 +4,8 @@
   "description": "An authentication plugin for Fastify",
   "main": "plugin.js",
   "scripts": {
-    "test": "tap 'test/**/*.test.js'",
-    "test-ci": "tap --cov 'test/**/*.test.js'",
```

Comment:
These `'` need to be present to prevent shell expansion of the globs.

Reply:
done I'm using quotes